config/services.yaml

```diff
@@ -3,13 +3,15 @@ parameters:
   env(SWAGGER_MOCK_CACHE_STRATEGY): 'disabled'
   env(SWAGGER_MOCK_CACHE_DIRECTORY): '/dev/shm/openapi-cache'
   env(SWAGGER_MOCK_CACHE_TTL): 0
+  env(SWAGGER_MOCK_CORS_ENABLE): False
```

Comment:
Please, add info about this parameter to README https://github.com/swagger-mock/swagger-mock#configuration

Reply:
Done!